xml/docu_styleguide.language.xml

```diff
@@ -468,16 +469,17 @@
   <para>
    Use quotations to quote from sources, such as books.
    In other cases, avoid using quotation marks.
-   Do not use irony and do not use quotation marks to mark irony.
-   In the case of computer input, computer output, and user
-   interface elements, use more appropriate markup. See also
+   For computer input, computer output, and user
+   interface elements, use different markup. See also
    <xref linkend="sec-command-line"/> and
    <xref linkend="sec-ui-item-markup"/>.
+   Also, do nor use quotations marks to mark irony. Avoid irony in
+   technical writing.
```

Comment:
Tbh, I don't see the advantage in these changes. It also moves the references to other sections in the middle of the paragraph rather than keeping them at the end, possibly making people click them before finishing the paragraph.

I do agree that "Do not use irony and do not use quotation marks to mark irony." sounds wooden and weird, however.

Reply:
Code suggestions are welcome!

Reply:
> Also, do nor use

Do _not_ use.